benchmark/multi_gpu/training/README.md

```diff
@@ -10,15 +10,42 @@ python training_benchmark_cuda.py --dataset ogbn-products --model edge_cnn --num
 
 ## Running benchmark on Intel GPU
 
-## Environment setup
+### Environment setup
 
+### Prerequisites
+
+- Intel Data Center GPU Max Series. You could try it through [Intel DevCloud](https://www.intel.com/content/www/us/en/developer/tools/devcloud/services.html).
+- Verify the Intel GPU Driver is installed, refer to the [guide](https://dgpu-docs.intel.com/driver/installation.html).
+
+### docker setup
+
+If you want to run your scripts inside a docker image, you could refer to the [dockerfile](https://github.com/pyg-team/pytorch_geometric/blob/master/docker/Dockerfile.xpu) and the corresponding [guide](https://github.com/pyg-team/pytorch_geometric/blob/master/docker).
+
+### bare-mental setup
+
+If you prefer to run your scripts directly on the bare-mental server. We recommend this [guide](https://intel.github.io/intel-extension-for-pytorch/xpu/latest/tutorials/features/DDP.html) and summarize it as follows:
```

Comment:
As above, `bare-mental` -> `bare-metal`.

Also, mentioned tutorial covers only oneCCL, we need for example MKL as well. I personally prefer instructions on the [main IPEX page](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bxpu&os=linux%2fwsl2&package=pip).

Reply:
Fixed and Added the link.